packages/aws-cdk-lib/aws-dynamodb/lib/global-table-base.ts

```diff
@@ -0,0 +1,488 @@
+import { DynamoDBMetrics } from './dynamodb-canned-metrics.generated';
+import * as perms from './perms';
+import { Operation, SystemErrorsForOperationsMetricOptions, OperationsMetricOptions, ITable } from './shared';
+import { IMetric, MathExpression, Metric, MetricOptions, MetricProps } from '../../aws-cloudwatch';
+import { Grant, IGrantable } from '../../aws-iam';
+import { IKey } from '../../aws-kms';
+import { Aws, Resource } from '../../core';
+
+export interface IGlobalTable extends ITable {
+  /**
+   * The ID of the Replica Table in the deployment region of the Global Table.
```

Comment:
Wut?

Reply:
I can change the wording here if it's confusing, but this is how all `GlobalTable` attributes work. The attribute be it `TableArn`, `TableStreamArn`, `TableId`, etc. will be for the `Replica` in the region that the stack containing the `GlobalTable`is deployed to. CloudFormation's exact wording is - "The TableId returned is that of the replica in the region the stack is deployed to". Would it be better to follow that verbiage?

Reply:
"The ID of the table in the main deployment region" ?

Reply:
I'll update the wording for all the attributes.